repo/packages/W/weavescope/0/marathon.json.mustache

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "/weavescope",
+  "cpus": 1,
+  "mem": 1024,
+  "instances": {{weavescope.probe.instances}},
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      }
+    ],
+    "docker": {
+      "image": "{{resource.assets.container.docker.da188a722491}}",
+      "network": "HOST",
+      "privileged": true,
+      "parameters": [
+        {
+          "key": "pid",
+          "value": "host"
+        }
+      ],
+      "forcePullImage": false
+    }
+  },
+  "args": [
+    "--probe.docker=true",
+    "weavescope.marathon.mesos"
+  ],
+  "ports": [
+    4040
+  ],
+  "requirePorts": true,
+  "labels": {
+    "DCOS_SERVICE_NAME": "weavescope",
+    "DCOS_SERVICE_SCHEME": "http",
+    "DCOS_SERVICE_PORT_INDEX": "1"
+  },
+  "acceptedResourceRoles": [ "public_slave", "*" ]
```

Comment:
@pyronicide somehow this is still not working as expected, it only gets to run on the private slaves...